docs/manual/java/guide/cluster/PersistentEntityCassandra.md

```diff
@@ -22,4 +22,54 @@ In sbt:
 
 ## Configuration
 
-Lagom's Cassandra support is provided by the [`akka-persistence-cassandra`](https://github.com/akka/akka-persistence-cassandra) plugin. A full configuration reference can be in the plugins [`reference.conf`](https://github.com/akka/akka-persistence-cassandra/blob/v0.20/src/main/resources/reference.conf).
+Lagom uses several internal Cassandra tables to persist entity data. You need to configure the keyspaces that are used for these tables. By default, Lagom automatically creates these keyspaces and tables at startup if they're missing.
```

Comment:
(1) I think this is ambiguous: `Lagom uses several internal Cassandra tables`. Lagom doesn't use "Cassandra's internal tables".

How about: "Lagom's Persistence needs a few tables to store it's data. These tables are stored in Cassandra keyspaces. A keyspace in Cassandra is a namespace that defines data replication on Cassandra nodes. Each service should use unique keyspace names so that the tables of different services do not conflict with each other. You need to configure the keyspaces that are used for these ..."
(rewrote first sentence and brought text from my other comment).